iam-drift-detection/parse_iam_from_tf_state_file.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+ORGANIZATION_ID="$1"
+GCS_STATEFILE="$2"
+
+TMPFILE=$(mktemp /tmp/XXXXX.tfstate)
+
+gcloud alpha storage cp $GCS_STATEFILE $TMPFILE
```

Comment:
Curious why you choose `alpha storage` instead of just `gsutil`?

Reply:
Not sure if they ever reached parity between gcloud storage and gsutil but last I heard gcloud storage is quite a bit more performant: https://groups.google.com/a/google.com/g/cloud-storage-discuss/c/kjq1cqmJZzY/m/WetDQFenAgAJ

Reply:
I switched to the GA version. I'm assuming these changes are live already.